features/steps/cli_steps.py

```diff
@@ -189,7 +189,7 @@ def create_config_file_with_tools(context, tools):
     It takes a custom tools list and sets example prompt to `y`.
     """
 
-    tools_str = tools if tools != "none" else ""
+    tools_str = tools
```

Comment:
Is this still necessary now we don't check for "none" ?

Reply:
tools == "" is not acceptable from config file, only "none/all" or short names, so I changed that line